### ISSUE TITLE
Publishing on Open VSX registry for VSCodium users

### ISSUE THREAD

**Issue by nicolas-brousse** [ruby/vscode-rdbg#5]
Do you plan to publish your extension to Open VSX registry?

VSCodium users are not using same registry as VSCode users.

https://github.com/eclipse/openvsx/wiki/Publishing-Extensions

**Comment by ko1** [comment]
I don't know enough knowledge about it.
Is it easy?

**Comment by nicolas-brousse** [comment]
I never did it already. I just ask once for someone else VSCode extension (https://github.com/benspaulding/vscode-procfile/issues/2).

They talk about a GitHub action too: https://github.com/HaaLeo/publish-vscode-extension#readme.

I can't tell more sorry 😔

**Comment by ko1** [comment]
Please reopen if there are "Open VSX registry" users who want to use this extension.

**Comment by arusa** [comment]
gitpod.com also needs the extension in the Open VSX registry:

> In that case, please reach out to the extension author and politely ask them to publish their extension to the vendor-neutral, open source Open VSX registry

https://www.gitpod.io/docs/ides-and-editors/vscode-extensions#where-do-i-find-extensions

They further link to this documentation about how to publish to Open VSX:

https://github.com/eclipse/openvsx/wiki/Publishing-Extensions

**Comment by raghubetina** [comment]
+1 for publishing on Open VSX — I teach hundreds of students per year using Gitpod, and they would greatly benefit from the improved debugging experience. Thank you!

**Comment by hksitorus** [comment]
+1

I also would like have this in my code-server
https://github.com/coder/code-server/blob/main/docs/FAQ.md#how-can-i-request-an-extension-thats-missing-from-the-marketplace

**Comment by ko1** [comment]
All right. So I need to google to publish it...

**Comment by arusa** [comment]
@ko1 check out the link I mentioned:

https://github.com/eclipse/openvsx/wiki/Publishing-Extensions

**Comment by HombreLaser** [comment]
+1, would be nice to have this extension in both vscodium and other microsoft independent forks

**Comment by ko1** [comment]
> +1, would be nice to have this extension in both vscodium and other microsoft independent forks

Are you using it?

**Comment by HombreLaser** [comment]
> > +1, would be nice to have this extension in both vscodium and other microsoft independent forks
> 
> Are you using it?

Yes! Both at work and in some school projects where I have the opportunity to use ruby. Is better than both pry-remote and plain rdbg and I enjoy debugging ruby code with it a lot.
Some linux distribution's repositories don't distribute microsoft's visual studio code (most notably, arch linux), and this extension is the the most notorious of the must-haves in a ruby development environment that's not published in the open vsx registry.

**Comment by ko1** [comment]
Thank you. I'll check it.

**Comment by stage-rl** [comment]
+1 Seems you have a solution ready, please implement. Thank you

**Comment by tanmoysrt** [comment]
+1 Many open-source projects [like [CircuitVerse](https://github.com/CircuitVerse/CircuitVerse/blob/master/SETUP.md)] added Gitpod to provide easy development flow for contributors, `vscode-rdbg` is great. As this is not available in Open-VSX , we can't use this extension for debugging in Gitpod or other open source VSCode alternative.
